### PR TITLE
FAQ ready for premier in production

### PR DESCRIPTION
**Pull request by Sakrecoer**

### CODE REVIEW

**Review by MaudSimon — APPROVED** [review]
LGTM

File: src/Frequently-Asked-Questions.md [hunk 1]
```diff
@@ -0,0 +1,126 @@
+# Frequently Asked Questions
+
+Before diving in, we recommend that you check the following ressources as they provide useful information about Alephium:
```
Comment (nop33) [inline src/Frequently-Asked-Questions.md:3]: Typo: ressources/resources

File: src/Frequently-Asked-Questions.md [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+- [Official Discord](https://discord.gg/JErgRBfRSB)
+- [Official Telegram](https://t.me/alephiumgroup)
+- [Official Medium](https://medium.com/@alephium), specifically: 
+    - [Alephium's Tokenomics](https://medium.com/@alephium/tokenomics-of-alephium-61d59b51029c)
+    - [Alephium's Block Rewards](https://medium.com/@alephium/alephium-block-rewards-72d9fb9fde33)
+    - [Alephium's Community Reward Program](https://medium.com/@alephium/introducing-community-rewards-f4638bbf14bf)
+    - [Alephium's Q1 2022 Update](https://medium.com/@alephium/alephium-q1-project-update-50f4a7b354b0)
+    - [Alephium's 1st AMA](https://medium.com/@alephium/alephiums-first-live-ama-761a90d3f672)
+
+FAQ
```
Comment (nop33) [inline src/Frequently-Asked-Questions.md:15]: Probably you want to make this an H2?
```
## FAQ
```

File: src/Frequently-Asked-Questions.md [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+Total Block Reward = Mining Reward + min(max(MR, 1 ALPH), Transaction Fee / 2)
+
+You will find a more elaborate explanation of the Block Reward [in this article from the Official Alephium Medium](https://medium.com/@alephium/alephium-block-rewards-72d9fb9fde33)
+
+### I have GPU model XYZ, what is my profit per day?
+A community member has made a spreadsheet with different GPUs [available here](https://docs.google.com/spreadsheets/d/10eUjwGU-Kmw1XM1dDOKfdscOeShakSnjcBGzBT46rmc/)
+
+### What is the reason to hold the block reward for 500 minutes, given the block time is only 64 seconds?
+
+The 500min lock was implemented to prevent re-org attacks. Similarly to Bitcoin, which has ~1000 minutes lock for mined rewards.
```
Comment (nop33) [inline src/Frequently-Asked-Questions.md:54]: Typo: Similarly/Similar

File: src/Frequently-Asked-Questions.md [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+
+### Why do I have 4 mining addresses?
+
+Alephium is a sharded blockchain with G groups and G*G shards. Due to this design, it is necessary to have one mining address per group. 
+
+On the mainnet, we currently have 4 Groups and 16 shards. It's why you have 4 mining addresses, one for each group.
+
+### Is there a group for alph miners?
```
Comment (nop33) [inline src/Frequently-Asked-Questions.md:62]: alph, ALPH, or Alephium? :/

File: src/Frequently-Asked-Questions.md [hunk 5]
```diff
@@ -0,0 +1,126 @@
...
+On [the Discord,](https://discord.gg/JErgRBfRSB) there is a special mining channel, where you can find active groups. 
+
+## dApps
+
+### What are stateful UTXOs and how are they different from the other UTXOs models?
+There are two type of states: mutable state (e.g. ETH) and immutable state (e.g. UTXO, Extended UTXO). Mutable state is much more expressive as you can see from the ecosystem of ETH, while eUTXO can be used to build some applications with limitations.
```
Comment (nop33) [inline src/Frequently-Asked-Questions.md:68]: Typo: There are two *types* of states

File: src/Frequently-Asked-Questions.md [hunk 6]
```diff
@@ -0,0 +1,126 @@
...
+
+## dApps
+
+### What are stateful UTXOs and how are they different from the other UTXOs models?
+There are two type of states: mutable state (e.g. ETH) and immutable state (e.g. UTXO, Extended UTXO). Mutable state is much more expressive as you can see from the ecosystem of ETH, while eUTXO can be used to build some applications with limitations. 
+In our stateful UTXO model, we support ETH-like mutable states. It allows us to easily build dApps as powerful as on Ethereum without the security concerns of the account model. 
+
+### Is anyone already building a DEX on ALPH?
+We have a simple [Uniswap-like DEX in the test](https://github.com/alephium/alephium/blob/master/app/src/it/scala/org/alephium/app/SmartContractTest.scala#L142-L170)
+We could also support order-book style DEX, which would avoid the well known impermanent loss problem thanks to the UTXO model.
```
Comment (nop33) [inline src/Frequently-Asked-Questions.md:73]: Typo: well known/well-known

File: src/Frequently-Asked-Questions.md [hunk 7]
```diff
@@ -0,0 +1,126 @@
...
+In our stateful UTXO model, we support ETH-like mutable states. It allows us to easily build dApps as powerful as on Ethereum without the security concerns of the account model. 
+
+### Is anyone already building a DEX on ALPH?
+We have a simple [Uniswap-like DEX in the test](https://github.com/alephium/alephium/blob/master/app/src/it/scala/org/alephium/app/SmartContractTest.scala#L142-L170)
+We could also support order-book style DEX, which would avoid the well known impermanent loss problem thanks to the UTXO model. 
+
+We identified DeFi and dApps to be the next critical focus for Alephium. To kickstart the development we will build clean Proof-of-concept dApps, to serve as examples. This will help us find bottlenecks or edge cases we haven’t been able to identify before. It will also serve as a basis to compile the necessary documentation to help community developers to build and deploy their applications. 
+Our VM and language are really dev-friendly and if you're familiar with Solidity, you can easily build similar application on Alephium.
```
Comment (nop33) [inline src/Frequently-Asked-Questions.md:76]: Typo: similar application/similar applications

File: src/Frequently-Asked-Questions.md [hunk 8]
```diff
@@ -0,0 +1,126 @@
...
+
+## Wallet
+
+### Is there a way to distinguish between a Bitcoin legacy address and an Alephium address?
+Alephium addresses are usually longer, as it uses 32 bytes hash instead of 20 bytes hash.
+
+## Tokenomics
+ 
+### If tokens are burnt, will there be a time in the future where the amount of existing ALPH will be close to zero?
+Theoretically yes, the future is not really projectable beyond 80 years. However it is not uncommon for open-source community-driven blockchains such as Alephium to change their policies along their evolution. If the community decides to alter a cap, then that cap will be altered.
```
Comment (nop33) [inline src/Frequently-Asked-Questions.md:86]: Typo: However/However,

File: src/Frequently-Asked-Questions.md [hunk 9]
```diff
@@ -0,0 +1,126 @@
...
+
+### What is the minimum Transaction Fee (TF)?
+Currently the minimum transaction fee is hard-capped at 0.00000000000001 ALPH. The pricing is however bound by auctioning and therefore dynamic. However we are investigating ways to alter the minimum TF dynamically based on hashrate.
```
Comment (nop33) [inline src/Frequently-Asked-Questions.md:89]: Typo: Currently/Currently,

Comment (nop33) [reply src/Frequently-Asked-Questions.md:89]: Typo: However/However,

File: src/Frequently-Asked-Questions.md [hunk 10]
```diff
@@ -0,0 +1,126 @@
...
+
+## Miscellaneous
+
+### How many Transactions Per Second (TPS)
+Currently up to 400 TPS with 16 shards. Alephium can scale up to 10k TPS by increasing the number of shards as necessary.
+
+### Why is the project named Alephium? 
+
+For those of you less familiar with set theory and mathematics the origin of the name “Alephium” might not be so evident. It is made from the name “Aleph” which is defined on wikipedia: “Aleph numbers are a sequence of numbers used to represent the cardinality of infinite sets that can be well-ordered. They were introduced by the mathematician Georg Cantor and are named after the symbol he used to denote them, the Hebrew letter aleph ( ℵ )”
```
Comment (nop33) [inline src/Frequently-Asked-Questions.md:98]: Typo: wikipedia/Wikipedia

File: src/Frequently-Asked-Questions.md [hunk 11]
```diff
@@ -0,0 +1,126 @@
...
+In an homage to the technical promises of Ethereum, we followed the same naming pattern and the name Alephium was coined.
+
+### Why did you choose PoLW, not PoS?
+
+Blockchain technology is still at an early stage and one of the most prevalent questions is: what blockchain infrastructure is needed for the next 10 years for Dapps, including DeFi ?
+
+We believe need a blockchain that is scalable with high throughput and low transaction fees. We want a high level of programmability like on Ethereum. And we need it to be as reliable and secure as Bitcoin is.  
+As a result, Alephium was developed on the idea to build a scalable Bitcoin with a reliable DeFi solution.
+
+According to the Lindy effect’s theory, and despite POS recent successes, it’s very likely that  the Bitcoin model and  sharding + PoW is still the most robust and decentralized way to build a scalable blockchain. Specifically:
```
Comment (nop33) [inline src/Frequently-Asked-Questions.md:111]: Double space at "that  the" and "and  sharding"

File: src/Frequently-Asked-Questions.md [hunk 12]
```diff
@@ -0,0 +1,126 @@
...
+
+1. PoW is simple and robust. It's much easier to design sharding algorithm with PoW
+2. PoS is not tested by time yet, we look forward to see how PoS will evolve after ETH's PoS switch
+3. PoS tends to be more centralized and more vulnerable to censure
+4. PoS tends to reduce trustlessness as the cost of running a node can be significantly higher
+5. PoS is more vulnerable to some DeFi attacks like MEV
+
+### Why not have 1M shards? 
+The groupsize G is not very big. Each node needs to maintain 2G - 1 other shards for consistency. We want to keep things small. 2G-1 cannot be too large. Given the average network bandwidth is enough, G can be set as high as 32. There are some computation overhead as well, but networking is the main bottleneck to push G higher.
```
Comment (nop33) [inline src/Frequently-Asked-Questions.md:120]: Typo: There are some computation overhead/There *is* some computation overhead

**Review by nop33 — COMMENTED** [review]
Sorry for checking this late, here are the typos I found

File: src/Frequently-Asked-Questions.md (hunk 4 above)
Comment (Sakrecoer) [reply src/Frequently-Asked-Questions.md:62]: Alephium :)

**Pull request merged**